spec/system/sdg_management/relations_spec.rb

```diff
@@ -88,6 +93,43 @@
       expect(page).to have_css "h2", exact_text: "Build a hospital"
     end
 
+    scenario "list records pending to review for the current model by default" do
+      create(:debate, title: "I'm a debate")
+      create(:sdg_review, relatable: create(:debate, title: "I'm a reviewed debate"))
+
+      visit sdg_management_debates_path
+
+      expect(page).to have_text "I'm a debate"
+      expect(page).not_to have_text "I'm a reviewed debate"
+
+      click_link "Pending"
```

Comment:
How about checking the "pending" filter has the active HTML class instead? `expect(page).to have_link "Pending", class: "is-active"`

The "problem" with clicking the link is that the texts we check are already present before clicking the link, and so there might be a potential race condition between the request and the expectations.

Reply:
Good catch! Done